### ISSUE TITLE
is it safe to use the version currently exist on the master branch

### ISSUE THREAD

**Issue by aymaneMx** [eamigo86/graphene-django-extras#134]
I faced this issue : 
```
  File "/opt/app/apps/community/graphql/inputs.py", line 2, in <module>
    from graphene_django_extras import DjangoInputObjectType
  File "/usr/local/lib/python3.7/site-packages/graphene_django_extras/__init__.py", line 4, in <module>
    from .directives import all_directives
  File "/usr/local/lib/python3.7/site-packages/graphene_django_extras/directives/__init__.py", line 48, in <module>
    all_directives = [d() for d in all_directives] + default_directives
  File "/usr/local/lib/python3.7/site-packages/graphene_django_extras/directives/__init__.py", line 48, in <listcomp>
    all_directives = [d() for d in all_directives] + default_directives
  File "/usr/local/lib/python3.7/site-packages/graphene_django_extras/directives/base.py", line 14, in __init__
    args=self.get_args(),
  File "/usr/local/lib/python3.7/site-packages/graphene_django_extras/directives/date.py", line 241, in get_args
    description='A format given by dateutil module',
TypeError: __init__() got an unexpected keyword argument 'type'
```

Than I noticed that you just merged a solution for that #132 

Is it safe to use this version on a production env?

**Comment by eamigo86** [comment]
Hi @aymaneMx, I just publish the new version with some minor fixes (including this one) in pypi (v0.4.9)

**Comment by aymaneMx** [comment]
thank you @eamigo86 🙏
BTW, thanks a lot for the great library.
